app/js/auth/index.js

```diff
@@ -7,7 +7,8 @@ import cookie from 'js-cookie';
 
 class Auth {
   andelaLoginUrl() {
-    return `${cookie.get(CVar.auth_url)}?redirect_url=http://${
+    const auth_url = cookie.get(CVar.auth_url) || 'http://api-staging.andela.com/login';
```

Comment:
I mean, there's a config folder that holds the several environments and configuration settings for each environment. The authentication URL can exist there.